src/map/water_supply.c

```diff
@@ -98,14 +102,19 @@ static void fill_aqueducts_from_offset(int grid_offset)
     int guard = 0;
     int next_offset;
     int image_without_water = image_group(GROUP_BUILDING_AQUEDUCT_NO_WATER);
+    int highway_image_with_water = assets_get_image_id("Logistics", "Highway_Aqueduct_Full_Left");
+    int highway_image_without_water = assets_get_image_id("Logistics", "Highway_Aqueduct_Empty_Left");
```

Comment:
Same here. In fact both variables could have file scope so you'd only need to fill the ids once instead of twice.